src/main/java/net/dries007/tfc/client/RenderHelpers.java

```diff
@@ -230,4 +267,16 @@ private static float calculateArmMovement(LocalPlayer player)
         }
         return 0f;
     }
+
+    public record RGBA(float r, float g, float b, float a)
+    {
+        public static RGBA of(int color)
+        {
+            final float r = ((color >> 16) & 0xFF) / 255F;
+            final float g = ((color >> 8) & 0xFF) / 255F;
+            final float b = (color & 0xFF) / 255F;
+            final float a = ((color >> 24) & 0xFF) / 255F;
+            return new RGBA(r, g, b, a);
+        }
+    }
```

Comment:
Isn't this record just a lightweight version of `java.awt.Color`, which we already use elsewhere?